courses/free-courses-pl.md

```diff
@@ -57,6 +57,7 @@
 
 ### Embedded
 
+* [Kurs Arduino, poziom II](https://www.forbot.pl/blog/kurs-arduino-ii-wstep-spis-tresci-id15494)
```

Comment:
Is Damian (Treker) Szymański the course creator?

Reply:
Yes, PR has been updated.